lib/stingray/toolchain.js

```diff
@@ -73,7 +73,7 @@ function preferredToolchainPath() {
 const readFile = Rx.Observable.fromNodeCallback(fs.readFile);
 export function configuration(toolchainPath) {
     var toolchainConfig = path.join(toolchainPath, "settings", "ToolChainConfiguration.config");
-    var userConfig = path.join(process.env.HOME, "AppData", "Local", "Autodesk", "Stingray", "Settings", "ToolChainConfiguration.config");
+    var userConfig = path.join(process.env.LOCALAPPDATA, "Autodesk", "Stingray", "Settings", "ToolChainConfiguration.config");
```

Comment:
@niklasfrykholm: When testing on my Windows 10 machine, I had to change this to get it to work. Seems the `HOME` environment variable is not always set. `LOCALAPPDATA` is supposed to be available from Windows Vista onward.